css/src/atomics/width.scss

```diff
@@ -1,7 +1,7 @@
 @use 'sass:list';
 
 $universal-widths: auto, 100, 150, 200, 250, 300, 350 !default;
-$mobile-incompatible-widths: 400, 450, unset !default;
+$mobile-incompatible-widths: 400, 450, 500, unset !default;
```

Comment:
This is a minor change for changeset. Reminder:

```
npx changeset add;
```
- select css package
- minor change
- add message

Reply:
reran changeset